hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/fs/ozone/TestRootedOzoneFileSystem.java

```diff
@@ -1038,4 +1043,41 @@ public void testRenameToTrashDisabled() throws IOException {
     ofs.delete(trashRoot, true);
   }
 
+  @Test
+  public void testFileDelete() throws Exception {
+    Path grandparent = new Path(bucketPath, "testBatchDelete");
+    Path parent = new Path(grandparent, "parent");
```

Comment:
Make this test parameterized to test with  "ozone.om.enable.filesystem.paths" enabled and disabled similar to TestOzoneFileSystem

Reply:
done